portfolio/src/main/java/com/google/sps/servlets/BestShadesServlet.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.servlets;
+                                
+import java.io.IOException;
+import java.util.ArrayList;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+                                                       
+@WebServlet("/shades")
+public class BestShadesServlet extends HttpServlet {
+    String listOfShades[] = {"Turquoise Blue", "Celeste", "Navy", "Cyan", "Light Turquoise"};
```

Comment:
I'd recommend going ahead and making this static final since you just create it once here. Something like:
`private static final String[] LIST_OF_SHADES = {"Turquoise Blue", "Celeste", "Navy", "Cyan", "Light Turquoise"};`

Also, we should include the [] with the variable type not the name, https://google.github.io/styleguide/javaguide.html#s4.8.3.2-array-declarations.